source/_integrations/google_assistant.markdown

```diff
@@ -75,7 +75,7 @@ If you've added Home Assistant to your phone's home screen, you have to first re
     6. For the Key type, select the JSON option.
     7. Click Create. A JSON file that contains your key downloads to your computer.
     8. Use the information in this file or the file directly to add to the `service_account` key in the configuration.
-3. If you didn't specify a service account and want to use the `google_assistant.request_sync` service, to update devices without unlinking and relinking, in Home Assistant, then enable HomeGraph API for your project:
+3. If you didn't specify a service account and want to use the `google_assistant.request_sync` service, to update devices without unlinking and relinking, in Home Assistant, then enable HomeGraph API for your project (Please note: this is now deprecated refer to Step 2 above, this feature will be removed in 0.105):
```

Comment:
Just change the "didnt" to "did" and drop the note  about deprecated.

Reply:
Ok will do, we also mention to note down the API key in step 3....should I remove that too (line 82)?

Reply:
Shouldn't we simply move Step 3.1 and Step 3.2 to be Step 2.9 and Step 2.10 and remove the rest of Step 3.

Reply:
Good suggestion! Updated now, I think this is better.  Let me know what you guys think.

Reply:
Maybe we could even merge the entire block "You will need to create a service account..." with Step 2. So the docs for "Service Account" and "Resync" would be at the same place.

Reply:
The reason why it is in 2 places is because of how easy it is for people to miss.  We should keep them there so people know to perform those steps.  There are many steps here and people tend to skim over it so mentioning it more than once is useful IMO.